content/docs/studio/user-guide/billing-and-payment.md

```diff
@@ -1,34 +1,37 @@
 # Billing and Payment
 
 Studio is free for individuals and teams with up to 2 members. The Free plan is
-designed for teams just starting out with collaboration: you can invite one
-other team member and import as many repositories as you want. For more advanced
-collaboration functionality, you can sign up for the Team or Enterprise plans.
-These are paid plans, and in this section you will learn how to manage billing
-and payment for these plans.
+designed for teams that are starting out: you can invite one other team member
+and import as many repositories as you want.
 
-- [Difference between Free, Team and Enterprise plans](#difference-between-free-team-and-enterprise-plans)
+For more advanced collaboration, you can sign up for the Team or Enterprise
+plans. These are paid plans, and in this page you will learn how to manage
+billing and payment for them.
+
+- [Difference between Free, Team, and Enterprise plans](#difference-between-free-team-and-enterprise-plans)
 - [How to sign up for the Team or Enterprise plan](#how-to-sign-up-for-the-team-or-enterprise-plan)
-- [How total price is calculated](#how-total-price-is-calculated)
-- [How increasing or decreasing the number of seats affects total cost](#how-increasing-or-decreasing-the-number-of-seats-affects-total-cost)
+- [How total price is calculated](#how-total-price-is-calculated), including
+  effects of making changes to the plan during the billing period.
 - [Downgrading a plan](#downgrading-a-plan)
-- [Billing period, auto-renewal of team plan, and grace period for non-payment](#billing-period-auto-renewal-of-team-plan-and-grace-period-for-non-payment)
+- [Billing period and auto-renewal of team plans](#billing-period-and-auto-renewal-of-team-plans)
+- [Grace period for non-payment](#grace-period-for-non-payment)
```

Comment:
Note that I separated these sections. a) Unclear that they're that closely related and b) the unified header was way too long (4 lines in the ToC) 😬 -- most headers here are already long...